justgage.js

```diff
@@ -231,6 +239,19 @@ JustGage = function(config) {
     pointerOptions: kvLookup('pointerOptions', config, dataset, [])
   };
```

Comment:
Is the `;` needed for`};` as it's not a `FunctionExpressions` with `obj.config = {`? 

You seem to be in the business of cleaning them up in other parts of this PR.